features/step_definitions/section_steps.rb

```diff
@@ -154,4 +154,5 @@
 
 Then('I can see elements from the section composed of a shadow root') do
   expect(@test_site.shadow_root.shadow_root_section).to have_some_text
+rescue SitePrism::UnsupportedGemVersionError
```

Comment:
Let's not run this scenario, and have it fail. Let's have it verbosely skipped.

Reply:
I've addressed it, https://github.com/site-prism/site_prism/pull/159/commits/a27523d9445dc5a0eaceb4547d22c06a40d0b15f